cos-gpu-installer-docker/gpu_installer_url_lib.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+get_major_version() {
+  echo "$1" | cut -d "." -f 1
+}
+
+get_minor_version() {
+  echo "$1" | cut -d "." -f 2
+}
+
+get_download_location() {
+  # projects/000000000000/zones/us-west1-a -> us
+  local -r instance_location="$(curl -sfS "http://metadata.google.internal/computeMetadata/v1/instance/zone" -H "Metadata-Flavor: Google" | cut -d '/' -f4 | cut -d '-' -f1)"
+  declare -A location_mapping
+  location_mapping=( ["us"]="us" ["asia"]="asia" ["europe"]="eu" )
+  # Use us as default download location.
+  echo "${location_mapping[${instance_location}]:-us}"
+}
+
+precompiled_installer_download_url() {
+  local -r driver_version="$1"
+  local -r major_version="$(get_major_version ${driver_version})"
+  local -r minor_version="$(get_minor_version ${driver_version})"
+  local -r milestone="$2"
+  # 11895.86.0 -> 11895-86-0
+  local -r build_id="${3//\./-}"
```

Comment:
I think this should be the third line (or fourth, with the comment) of the function, for the same reason as above

Reply:
Done